pages/housing-assistance/disability-housing-grants/how-to-apply.md

```diff
@@ -1,45 +1,38 @@
 ---
 layout: page-breadcrumbs.html
 template: detail-page
-title: How to Apply for a Specially Adapted Housing (SAH) Grant
-display_title: How to Apply
+title: How to apply for a Specially Adapted Housing (SAH) grant
+display_title: How to apply
 description: Find out how to apply for a Specially Adapted Housing (SAH) grant or a Special Home Adaptation (SHA) grant. These grants offer financial help so you can make changes to your home to live more independently with a service-connected disability. 
 concurrence:
 order: 1
 ---
 
+
 <div itemscope itemtype ="http://schema.org/HowTo">
 <div class="va-introtext" itemprop="description">
-
+ 
 Find out how to apply for a Specially Adapted Housing (SAH) or Special Home Adaptation (SHA) grant on our eBenefits website. You can also check the status of an open SAH claim on the site.
 
 </div>
 
-<div itemscope itemtype="http://schema.org/Question">
-<h2 itemprop="name">How do I prepare before starting my application?</h2>
-<div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
-<div itemprop="text">
-<ul>
+<div class="feature">
+ 
+ ## How do I prepare before starting my application?
 
-<li><a href="/housing-assistance/disability-housing-grants/">Find out if you’re eligible for an SAH or SHA grant</a>.</li>
+
+<li><a href="/housing-assistance/disability-housing-grants/">Find out if you’re eligible for an SAH or SHA grant</a></li>
```

Comment:
Should the `<li>` tags be here? Asking because this is failing accessibility tests -

```
Failed [fail]: (http://localhost:3001/housing-assistance/disability-housing-grants/how-to-apply/: [serious] <li> elements must be contained in a <ul> or <ol>
   See https://dequeuniversity.com/rules/axe/2.6/listitem?application=axeAPI
```

Reply:
@ncksllvn I added the missing ul opening and closing tags. Now it seems fine. Are you able to confirm and merge?

Reply:
That should do it!